src/components/node-select.vue

```diff
@@ -39,45 +38,48 @@ export default {
   i18nOptions: { namespaces: "common" },
   data() {
     return {
-      nodes: nodeList,
+      blockchain: getBlockchain(this.$store.state.WalletStore.wallet.chain),
```

Comment:
Since we're gonna switch to a master pass type setup and multiple chain accounts chosen via a dropdown or something,   chain and associated config list should be switchable. We can use event-bus to notify the node-select component of changes.

Reply:
node-select does not make much sense in a multi-chain environment.

I suggest to move it to a config slide and present all chains with connectivity options at once.

Reply:
I guess it depends on node reliability. Maybe the built-in list is overkill but allowing the user to specify a main node and a fallback node is not. + the connectivity indicator when switching from say a BTS account to a STEEM account is important so the functionality of the node-select component is important. 

Making it a different slide removes the status from the main view. Unless we make the node-select component expandable on-click (shows connectivity of selected account's chain and on expand shows all). But that means remaining connected to all chains on the background which probably requires more refactoring.

Reply:
Beet might be connected to several chains at the same time (it already is now through verified messages, with no problem).

I imagine having the chain logos in the footer and indicate connectivity through color. Then you have onclick expansion where you can choose nodes for each chain if desired